src/main/java/dev/pgm/community/sessions/services/SQLSessionService.java

```diff
@@ -0,0 +1,207 @@
+package dev.pgm.community.sessions.services;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import dev.pgm.community.database.DatabaseConnection;
+import dev.pgm.community.feature.SQLFeatureBase;
+import dev.pgm.community.sessions.Session;
+import dev.pgm.community.sessions.SessionQuery;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import javax.annotation.Nonnull;
+import tc.oc.pgm.util.concurrent.ThreadSafeConnection.Query;
+
+public class SQLSessionService extends SQLFeatureBase<Session, SessionQuery> {
+
+  private static final String TABLE_NAME = "sessions";
+  private static final String TABLE_FIELDS =
+      "(id VARCHAR(36) PRIMARY KEY, player VARCHAR(36), disguised BOOL, server VARCHAR(32), startTime LONG, endTime LONG)";
```

Comment:
Shouldn't this be `start_time` and `end_time`? I know you might be following "convention" (because friendships also use camel case instead of snake case), but still, I think it's far better to have it as snake case